doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -399,6 +400,18 @@
     }
     ```
 
+20. Renamed `BaseRelativePointer` to `UntypedRelativePointer`
+
+    ```cpp
+    // before
+    #include "iceoryx_hoofs/internal/relocatable_pointer/base_relative_pointer.hpp"
+    iox::rp::BaseRelativePointer myUntypedRelativePointer;
+
+    // after
+    #include "iceoryx_hoofs/memory/relative_pointer.hpp" /// @todo #605 adapt namespace and directory
```

Comment:
Does this mean that the relative ptr will be moved to this directory in a follow-up PR? When is this planned? Usually I prefer to document the current state and not the planned state if it's not clear when the code will be refactored.

Reply:
@FerdinandSpitzschnueffler I plan to do this right after this PR here is merged. I will prepare it in advance and link it here.

Reply:
Done in #1677